packages/zwave-js/src/lib/controller/Controller.ts

```diff
@@ -4309,6 +4397,14 @@ ${associatedNodes.join(", ")}`,
 	 * This will assign up to 4 routes, depending on the network topology (that the controller knows about).
 	 */
 	public async assignSUCReturnRoutes(nodeId: number): Promise<boolean> {
+		if (nodeId >= 0x100) {
+			this.driver.controllerLog.logNode(nodeId, {
+				message: `Skipping SUC return route because isLR...`,
+				direction: "outbound",
+			});	
+			return true;
+		}
+
```

Comment:
This should _probably_ be conditioned on a per-node flag/property like "supports routing" or something. I think that's actually specified/fetched during the interview process or something for nodes, yes?

Long range nodes don't mesh, so don't need routes, and have no SUC.